tests/tracer.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+
+	"github.com/weaveworks/tcptracer-bpf/pkg/event"
+	"github.com/weaveworks/tcptracer-bpf/pkg/tracer"
```

Comment:
There is no good reason, I guess...

At the beginning, this repository contained only the ebpf code (in C) without Golang code. The Golang code was in a separate repository. Then, we moved it together because it is tightly coupled and it is more practical to enable CI when it's together. When the Golang code was copied here, it was at first just in the "tests/" directory (and code duplicated in Scope). This was then moved partially to "pkg/" to allow Scope to reuse it without duplicating the code.

@peterbourgon 's blog post suggests to have everything either in "pkg/" and "cmd/" but not at the top-level:
https://peter.bourgon.org/go-best-practices-2016/#repository-structure
Should I merge the 4 pkgs into a single pkg in "pkg/tracer/"?